delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/acls/Acls.scala

```diff
@@ -0,0 +1,109 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.acls
+
+import ch.epfl.bluebrain.nexus.delta.sdk.AclResource
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Caller
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclRejection.RevisionNotFound
+import monix.bio.{IO, Task}
+
+/**
+  * Operations pertaining to managing Access Control Lists.
+  */
+trait Acls {
+
+  /**
+    * Fetches the ACL resource for a ''target'' on the current revision.
+    *
+    * @param target the target location for the ACL
+    */
+  def fetch(target: Target): Task[Option[AclResource]]
+
+  /**
+    * Fetches the ACL resource for a ''target'' on the passed revision.
+    *
+    * @param target the target location for the ACL
+    * @param rev    the revision to fetch
+    */
+  def fetchAt(target: Target, rev: Long): IO[RevisionNotFound, Option[AclResource]]
```

Comment:
Do we need an option here in the result type? I understand that it allows to discriminate between the "resource is not found" and "resource exists but the revision doesn't ", but in terms of responses you'd still return a 404.
I went the other way, but if you think this is better, I will adjust the contract for realms.

Reply:
Since the response payload is going to be different, we should distinguish them